specs/_features/maxeb_increase/capella.py

```diff
@@ -1776,6 +1790,49 @@ def process_execution_layer_withdraw_request(
     ))
 
 
+def is_consolidated(validator: Validator) -> bool:
+   validator.consolidated_to != UNSET_CONSOLIDATED_TO
+
+
+def process_consolidation(state: BeaconState, consolidation: Consolidation) -> None:
+    target_validator = state[consolidation.target_index]
+    source_validator = state[consolidation.source_index]
+
+    assert is_active_validator(target_validator) and not target_validator.slashed and not is_consolidated(target_validator)
```

Comment:
All these conditions for the target validator may not be required but look sensible on a first approach